src/components/templates/PosterPage/components/IntroVideoPreviewModal/IntroVideoPreviewModal.tsx

```diff
@@ -30,7 +28,9 @@ export const IntroVideoPreviewModal: React.FC<IntroVideoPreviewModalProps> = ({
       <iframe
         className="IntroVideoPreviewModal__iframe"
         title="IntroVideoPreviewModal-iframe"
-        src={posterVenue?.poster?.introVideoUrl}
+        src={introVideoUrl}
+        allow={IFRAME_ALLOW}
+        allowFullScreen
       />
```

Comment:
[OOS] OOS for this PR, but we should extract a generic iframe/embeddable type component so that we don't have to keep manually remembering to set all of the various allow/etc props we consistently use.

Reply:
We have that extracted in `new-auditorium` PR

Reply:
Awesome!